bundles/org.openhab.transform.javascript/src/main/java/org/openhab/transform/javascript/internal/profiles/JavaScriptTransformationProfile.java

```diff
@@ -52,7 +51,7 @@
     @NonNullByDefault({})
```

Comment:
This annotation is wrong. l. 75 sets it to `null`. (also above)

Reply:
Absolutely. I changed it.